api/lib/domain/usecases/certificate/get-certification-attestations-for-session.js

```diff
@@ -0,0 +1,33 @@
+const { NotFoundError } = require('../../errors');
+const { isEmpty, compact } = require('lodash');
+const bluebird = require('bluebird');
+
+module.exports = async function getCertificationAttestationsForSession({
+  sessionId,
+  certificateRepository,
+  certificationCourseRepository,
+}) {
+  const certificationCourses = await certificationCourseRepository.findCertificationCoursesBySessionId({ sessionId });
+
+  if (isEmpty(certificationCourses)) {
+    throw new NotFoundError();
+  }
+
+  const certificationAttestations = compact(
+    await bluebird.mapSeries(certificationCourses, async (certificationCourse) => {
+      try {
+        return await certificateRepository.getCertificationAttestation(certificationCourse.getId());
+      } catch (error) {
+        if (!(error instanceof NotFoundError)) {
+          throw error;
+        }
+      }
+    })
+  );
```

Comment:
Question d'ordre générale, est-ce que à terme il est prévu de se passer de Bluebird sur Pix ? Pour un passage en full ES6 ?
Par exemple ici l'usage de bluebird pourrait être évité pour une version sans framework.

```suggestion
   const getAttestations = compact(await Promise.all([...certificationCourses.map(certificationCourse => async (certificationCourse) => {
      try {
        return await certificateRepository.getCertificationAttestation(certificationCourse.getId());
      } catch (error) {
        if (!(error instanceof NotFoundError)) {
          throw error;
        }
      }
    })]);
  );

```